### PR TITLE
[ISSUE-#12] added basic layout for the blog page

### PR DESCRIPTION
**Pull request by MaxwellKJr**
Made a basic layout for the blog page/section. Wasn't sure about the color red for the card background.

### CODE REVIEW

**Comment by joelfickson** [comment]
This commit is failing. Can you try `npm run build`

File: pages/blog/BlogCard.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			<Link href={`blog/${post.slug}`}>
+				<a className="inline-block p-2 bg-malawi-black transition-all hover:bg-malawi-white hover:text-malawi-black-dark rounded-sm transform hover:translate-x-2 hover:-translate-y-1 hover:shadow-lg shadow-malawi-black">
+					Read More
+				</a>
+			</Link>
+		</div>
+	</div>
+);
+
+export default BlogCard;
```
Comment (joelfickson) [inline pages/blog/BlogCard.js]: Can you make sure you are using our prettier and eslint styling.

File: pages/blog/BlogCardsContainer.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import BlogCard from './BlogCard';
```
Comment (joelfickson) [inline pages/blog/BlogCardsContainer.js]: Also here, use our prettier.

**Review by joelfickson — CHANGES_REQUESTED** [review]
Please address the comments. 

Plus try to run `npm run build` locally. You will see it will not build because `slug` is undefined.

File: pages/blog/BlogCard.js (hunk 1 above)
Comment (MaxwellKJr) [reply pages/blog/BlogCard.js]: This is currently the only issue I have. I'm using the same prettierrc.json and eslintrc.json that's in the repo but they are not taking effect. It's not even the formatting I use myself. Is there a way I can force it to use those files?

Comment (joelfickson) [reply pages/blog/BlogCard.js]: What editor are you using? If you are on VsCode I think you can save the prettier configs there.

Comment (MaxwellKJr) [reply pages/blog/BlogCard.js]: I'm using vs code. I will try to change a few things and will send a screenshot of how it's formatting.

**Pull request closed without merging**